CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 ### 2.3.2 (TBD)
 
-- Bug: On a MacOS, files generated under /etc/resolver as the result of using include-suffixes in the cluster config, are now properly removed on quit.
+- Change: The list of subnets that the virtual network interface will route is configured dynamically and will follow changes in the cluster.
```

Comment:
is there a docs PR w/ release notes for all these changes + any other docs changes? Or did you / will you not have time for that?

Reply:
Answered offline, Thomas plans to do that tomorrow!